### ISSUE TITLE
Release tag

### ISSUE THREAD

**Issue by 13bscsaamjad** [esrf-bliss/Lima#112]
Hi,

Can someone please update the version with the release tag. There was a fix #109 for hdf5 multiset writing merged into master with commit cce35de33c4a03c269c789697db87238bc398cb1. I suggest 1.9.2 or whatever is correct according to the guidelines of the project.

The updated version is required to be built and deployed here at MAX IV and it would be great if there is a release tag so our build is in sync with the upstream.

Thanks a lot in advance.

Best,
Abdullah

**Comment by laurent-claustre** [comment]
Dear Abdullah,
we are in a middle of preparing new versions of Lima. We recently decided to update the LIMA H5 file structure to be more compliant wit Nexus standard and for a better integration with data analysis pipeline here at ESRF.   Could you tell us if the new structure is suitable for you or not MAX IV?
If not Ok for you we should prepare a intermediate version 1.9.2 with only  #109, and then the version 1.9.3 will follow with the new H5 structure and some other stuffs.

**Comment by 13bscsaamjad** [comment]
Dear Laurent,
Thanks for your reply. I think it is better if it is possible to prepare a version with only #109 for now. The current structure is being used here and everyone is familiar with it as of now. I am sure the Nexus standard will have many advantages but probably it will require some time here at MAX IV to get acquainted to or at least get familiar with.

**Comment by laurent-claustre** [comment]
Dear Abdullah, 
did you already deploy  v1.9.2? If not we would like to move the tag to fix here a CI issue to build the conda packages.

**Comment by 13bscsaamjad** [comment]
Dear Laurent,
You can move on; I will checkout on the tag and build.

**Comment by laurent-claustre** [comment]
We found a serious bug when stopping an acquisition before the end, you will get the last h5 file not properly closed and then corrupted. Did you observe this issue?

Are you using the conda packages or are you compiling Lima from scratch?

**Comment by 13bscsaamjad** [comment]
I am building from scratch, I have not found this because I only built and did not deploy and validate yet.

**Comment by laurent-claustre** [comment]
No,  the bug exists since years, so you could encountered the same issue.
Could you for information tell me for which detector and how many you are using Lima, please?

**Comment by 13bscsaamjad** [comment]
As of now, we use Lima for all the Baslers here, thats probably hundred of them I guess, Merlin, Andor Xyla, Andor Balor, Tucsen Dhyana (in production), Raptor and Prosilica. We used it for Pilatus as well but not anymore.

**Comment by laurent-claustre** [comment]
thanks for the info. I never heard about Tucsen Dhyana nor Raptor, do you plane to make them available for the community on github.com?

**Comment by 13bscsaamjad** [comment]
yes, I have to test the dhyana plugin, i will connect that to this project when its done. i will check for raptor as well.
